File: WebApp/wwwroot/scripts/domain/emailRecipient.ts
module App.Models {
    "use strict";

    export interface IEmailRecipient extends ng.resource.IResource<IEmailRecipient> {
        id: string;
        orgUserId: Models.IOrgUser;
        orgUserName: string;
        email: string;
        feedbacks: boolean;
        newMobileUsers: boolean;
        orgRequests: boolean;
        orgConnectionRequests: boolean;
    }
}

File: Models/Entities/EmailRecipient.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace LightMethods.Survey.Models.Entities
{
    public class EmailRecipient : Entity
    {
        [Required]
        [Index("IX_User_Recipient", 1, IsUnique = true)]
        public Guid OrgUserId { get; set; }

        public virtual OrgUser OrgUser { get; set; }

        public bool Feedbacks { get; set; }

        public bool NewMobileUsers { get; set; }

        public bool OrgRequests { get; set; }

        public bool OrgConnectionRequests { get; set; }
    }
}
